build/match_byte.rs

```diff
@@ -27,20 +29,18 @@ pub fn expand(from: &Path, to: &Path) {
 
 struct MatchByte {
     expr: syn::Expr,
-    arms: Vec<syn::Arm>,
+    _comma: Token![,],
+    arms: Punctuated<syn::Arm, Option<Token![,]>>,
```

Comment:
The `Arm` parser will always eat a trailing comma if there is one. https://github.com/dtolnay/syn/blob/0.15.12/src/expr.rs#L2777-L2781. The comma will go in https://docs.rs/syn/0.15/syn/struct.Arm.html#structfield.comma.

I would recommend parsing these as Vec\<Arm\> as seen in https://github.com/dtolnay/syn/blob/0.15.12/src/expr.rs#L1898-L1901.